> This currently does not build. CC @illdefined
> 
> ```
>       > Executing pythonImportsCheckPhase
>        > Check whether the following modules can be imported: pyprland pyprland.adapters pyprland.adapters.menus pyprland.command pyprland.common pyprland.ipc pyprland.plugins pyprland.plugins.experimental pyprland.plugins.expose pyprland.plugins.fetch_client_menu pyprland.plugins.interface pyprland.plugins.layout_center pyprland.plugins.lost_windows pyprland.plugins.magnify pyprland.plugins.monitors pyprland.plugins.monitors_v0 pyprland.plugins.pyprland pyprland.plugins.scratchpads pyprland.plugins.shift_monitors pyprland.plugins.shortcuts_menu pyprland.plugins.system_notifier pyprland.plugins.toggle_dpms pyprland.plugins.toggle_special pyprland.plugins.workspaces_follow_focus
>        > Traceback (most recent call last):
>        >   File "<string>", line 1, in <module>
>        >     import sys; import importlib; list(map(lambda mod: importlib.import_module(mod), sys.argv[1:]))
>        >                                   ~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
>        >   File "<string>", line 1, in <lambda>
>        >     import sys; import importlib; list(map(lambda mod: importlib.import_module(mod), sys.argv[1:]))
> ```

Did you perhaps intend to ping @iliayar?

Otherwise I’ll gladly try to support, but it might take a while with my current workload.

--- a/pkgs/by-name/py/pyprland/package.nix
+++ b/pkgs/by-name/py/pyprland/package.nix
@@ -7,21 +7,23 @@
 
 python3Packages.buildPythonApplication (finalAttrs: {
   pname = "pyprland";
-  version = "2.6.2";
+  version = "3.1.0";
   pyproject = true;
 
   src = fetchFromGitHub {
     owner = "hyprland-community";
     repo = "pyprland";
     tag = finalAttrs.version;
-    hash = "sha256-jvdXytMnNrINNkSNljYFS9uomPmQ0g2Bnje/8YbsAv8=";
+    hash = "sha256-qayntjpE3WYBl6idfpwtvkEo61oHtc80uNEm0m4XA2o=";
   };
 
   nativeBuildInputs = with python3Packages; [ poetry-core ];
 
   propagatedBuildInputs = with python3Packages; [
     aiofiles
+    aiohttp
     pillow
+    questionary
   ];
   pythonRelaxDeps = [
     "aiofiles"
@@ -51,7 +53,6 @@ python3Packages.buildPythonApplication (finalAttrs: {
     "pyprland.plugins.lost_windows"
     "pyprland.plugins.magnify"
     "pyprland.plugins.monitors"
-    "pyprland.plugins.monitors_v0"
     "pyprland.plugins.pyprland"
     "pyprland.plugins.scratchpads"
     "pyprland.plugins.shift_monitors"
@@ -66,7 +67,7 @@ python3Packages.buildPythonApplication (finalAttrs: {
 
   meta = {
     mainProgram = "pypr";
-    description = "Hyperland plugin system";
+    description = "Hyprland plugin system including scratchpads";
     homepage = "https://github.com/hyprland-community/pyprland";
     license = lib.licenses.mit;
     maintainers = with lib.maintainers; [

